controllers/healthcheck_controller.go

```diff
@@ -107,6 +109,31 @@ func (r *HealthCheckReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error)
 		}
 		return ctrl.Result{}, ignoreNotFound(err)
 	}
+
+	return r.execHealthCheck(ctx, req, log, healthCheck)
+}
+
+func (r *HealthCheckReconciler) execHealthCheck(ctx context.Context, req ctrl.Request, log logr.Logger, healthCheck *activemonitorv1alpha1.HealthCheck) (ctrl.Result, error) {
+	defer func() {
+		if err := recover(); err != nil {
+			log.Info("Error: Panic occurred during execAdd %s/%s due to %s", healthCheck.Name, err)
```

Comment:
@RaviHari, looks like the formatting string may expect 3 params to interpolate however, only name and err are then being passed. Is the thought that you want to see the name and namespace on either side of the `/`? If so, I'll offer a quick fix below.

Though, arguably that info may be redundant since the `log` object has already been configured (line 91) to include namespace+name at the head of each log line.

```suggestion
			log.Info("Error: Panic occurred during execAdd %s/%s due to %s", healthCheck.Name, healthCheck.Namespace, err)
```

Reply:
Yes thats right.. Thanks for finding this..